City-of-Helsinki/berth-reservations-admin
# VEN-855 | Implement stickers for unmarked WS notices

## Description :sparkles:

Adds sticker information to unmarked WS notices with connected leases.

### Closes :no_good_woman:

[VEN-855](https://helsinkisolutionoffice.atlassian.net/browse/VEN-855)

## Testing :alembic:

### Automated tests :gear:️

- `src/common/unmarkedWsNoticeDetails/fragments/stickerDetails/__tests__/StickerDetails.test.tsx`

### Manual testing :construction_worker_man:

- Go to unmarked WS notices page
- Open details rows of the notices
  - If the notice has an attached lease, it should contain sticker details
  - If the notice is paid, it should be possible to change the sticker number
- Same information should be present in a single notice's page
